test/system/800-config.bats

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env bats   -*- bats -*-
+#
+# Test specific configuration options and overrides
+#
+
+load helpers
+
+@test "podman CONTAINERS_CONF - CONTAINERS_CONF in conmon" {
+    skip_if_remote "can't check conmon environment over remote"
+
+    # Make an innocuous containers.conf in a non-standard location
+    conf_tmp="$PODMAN_TMPDIR/containers.conf"
+    cat >$conf_tmp <<EOF
+[engine]
+runtime="${OCI_RUNTIME:-crun}"
```

Comment:
Just to save you the time of waiting for CI to finish: this is likely to fail on one of our CI hosts, in which only `runc` is installed. You may want to get the default from `podman info` instead.

Reply:
Good idea, implemented.